chain/chunks/src/lib.rs

```diff
@@ -754,6 +812,11 @@ impl ShardsManager {
 
         // 7. Checking epoch_id validity
         let prev_block_hash = header.inner.prev_block_hash;
+        if self.runtime_adapter.get_epoch_id_from_prev_block(&prev_block_hash).is_err() {
```

Comment:
Use `match` on the call below instead? Seems weird to call it twice :)

```
let epoch_id = match self.runtime_adapter.get_epoch_id_from_prev_block(&prev_block_hash) {
    Ok(epoch_id) =>  epoch_id,
    Err(_) => return Ok(ProcessPartialEncodedChunkResult::NeedBlock),
};
```

Reply:
++